build.gradle

```diff
@@ -23,7 +23,7 @@ def TEMPLATE_LIST_FILE_NAME = "templates.manifest"
 task buildGitCommitFile {
 	doLast {
 		def commitId = grgit.head().id
-		// is there a varible for builddir/classe/java/main?
+		// is there a variable for builddir/classe/java/main?
```

Comment:
typo: `classe`

Reply:
Yep, already fixed locally from the last PR. It'll show up in 2 PRs